profile/README.md

```diff
@@ -7,110 +7,110 @@ Unnofficial Supabase libraries, examples, and repositories by the community for
 
 <table style="table-layout:fixed; white-space: nowrap;">
   <tr>
-    <th>Language</th>
+    <td>Language</td>
     <th>Client</th>
     <th colspan="4">Feature-Clients (bundled in Supabase client)</th>
   </tr>
   <tr>
-    <th></th>
-    <th>Supabase</th>
-    <th><a href="https://github.com/postgrest/postgrest" target="_blank" rel="noopener noreferrer">PostgREST</a></th>
+    <td></td>
+    <th><a href="https://github.com/supabase/supabase" target="_blank" rel="noopener noreferrer">Supabase</a></th>
     <th><a href="https://github.com/supabase/gotrue" target="_blank" rel="noopener noreferrer">GoTrue</a></th>
+    <th><a href="https://github.com/postgrest/postgrest" target="_blank" rel="noopener noreferrer">PostgREST</a></th>
```

Comment:
We're kind of using a "historical" ordering here. So at the start was supabase + postgrest, then we added auth, then realtime, then storage, etc.

Reply:
I understand now 👍🏼 👌🏼